libcpu/arc/em/contex_gcc_mw.S

```diff
@@ -4,20 +4,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 #define __ASSEMBLY__
```

Comment:
contex_gcc_mw.S, which used for GCC based toolchain?

Reply:
file contex_gcc_mw.S is used for gcc toolchain and metaware(mw) toolchain both.